aicsimageio/tests/metadata/test_utils.py

```diff
@@ -6,46 +6,39 @@
 
 from aicsimageio import AICSImage
 
-from ..conftest import get_resource_full_path, host
+from ..conftest import get_resource_full_path, LOCAL
 
 ###############################################################################
 
 
-@host
 @pytest.mark.parametrize(
     "filename",
     [
         # DefaultReader
         pytest.param(
             "example.png",
-            marks=pytest.mark.raises(execeptions=NotImplementedError),
+            marks=pytest.mark.raises(exception=NotImplementedError),
         ),
         # TiffReader
         pytest.param(
             "s_1_t_10_c_3_z_1.tiff",
-            marks=pytest.mark.raises(execeptions=NotImplementedError),
+            marks=pytest.mark.raises(exception=NotImplementedError),
         ),
         # OmeTiffReader
-        ("actk.ome.tiff"),
+        "actk.ome.tiff",
         # LifReader
         pytest.param(
             "tiled.lif",
-            marks=pytest.mark.raises(execeptions=NotImplementedError),
+            marks=pytest.mark.raises(exception=NotImplementedError),
         ),
         # CziReader
-        pytest.param(
-            "s_1_t_1_c_1_z_1.czi",
-            marks=pytest.mark.raises(execeptions=NotImplementedError),
-        ),
-        pytest.param(
-            "RGB-8bit.czi",
-            marks=pytest.mark.raises(execeptions=NotImplementedError),
-        ),
+        "s_1_t_1_c_1_z_1.czi",
+        "RGB-8bit.czi",
     ],
 )
-def test_ome_metadata(filename: str, host: str) -> None:
+def test_ome_metadata(filename: str) -> None:
     # Get full filepath
-    uri = get_resource_full_path(filename, host)
+    uri = get_resource_full_path(filename, LOCAL)
```

Comment:
Running these tests against `REMOTE` doesn't work right now, and I don't think it's necessary anyways.

Reply:
fair and agreed.